FBKVOController/FBKVOController.h

```diff
@@ -9,6 +9,33 @@
 
 #import <Foundation/Foundation.h>
 
+/**
+ This macro ensures that key path exists at compile time.
+ Given a real receiver with a key path as you would call it, it verifies at compile time that the key path exists, without calling it.
+ 
+ For example:
+
+ FBKVOKeyPath(string.length) => @"length"
+
+ Or even the complex case:
+ 
+ FBKVOKeyPath(string.lowercaseString.length) => @"lowercaseString.length".
+ */
+#define FBKVOKeyPath(KEYPATH) \
+@(((void)(NO && ((void)KEYPATH, NO)), strchr(# KEYPATH, '.') + 1))
```

Comment:
This is SUPER unsafe code. If the key path doesn't contain `.`, then `strchr` returns NULL, and we pass address 0x0001 as the key path instead of the original key path, which probably isn't intended. You can refactor this using statement expressions if you'd like.